.env.virtual-funding

```diff
@@ -0,0 +1,18 @@
+## shared between multiple packages
```

Comment:
Why add a new monorepo env file?

Reply:
It allows us to do `SC_ENV=virtual-funding yarn start` (there are no env files checked into this repo at present).

Reply:
What's the objective though? I'm not aware of any integration test which uses multiple services in the repo atm

Reply:
I propose that we _should_ have an integration test in this repo (see my other comment).  If you like I can save the addition of this file until I make that PR?

Reply:
Progress on such a test over at #2437